Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -521,6 +584,6 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
 
     _handleFontVariationValueChanged(event)
     {
-        this._fontStyles.writeFontVariation(event.data.tag, event.data.value);
+        this._fontStyles.writeFontVariation(event.data.tag, parseFloat(event.data.value));
```

Comment:
Is the value ever not a float? Seems like an issue?

Reply:
You're right, the value should be a float, but handled in another event handler.

This event handler receives data verbatim from the "input" event handler for the `<input type="range">` defined in `WI.FontVariationDetailsSectionRow`. That's a DOM event. The property values are strings. We should convert the string to float in that handler, not here.